<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLaplacian</class>
 <widget class="QDialog" name="DialogLaplacian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>size</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="sliderSize">
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>scale</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="sliderScale">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>delta</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="sliderDelta">
     <property name="maximum">
      <number>25500</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>border</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cBxBorder">
     <item>
      <property name="text">
       <string>constant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>replicate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>reflect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>reflect_101</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
